gradio/processing_utils.py

```diff
@@ -55,10 +57,15 @@ def extract_base64_data(x: str) -> str:
 def decode_base64_to_image(encoding: str) -> Image.Image:
     image_encoded = extract_base64_data(encoding)
     img = Image.open(BytesIO(base64.b64decode(image_encoded)))
-    exif = img.getexif()
-    # 274 is the code for image rotation and 1 means "correct orientation"
-    if exif.get(274, 1) != 1 and hasattr(ImageOps, "exif_transpose"):
```

Comment:
Why did we remove this check `exif.get(274, 1) != 1`

Reply:
As the commit message implies, `exif_transpose` does it by itself.

https://github.com/python-pillow/Pillow/blob/3ffa8dca446ba22f3f143331e86cfef0e9caf8f2/src/PIL/ImageOps.py#L593-L602